src/utilities.cpp

```diff
@@ -37,5 +37,34 @@ void write_file_to_flash(const char *filename, uint32_t* counter) {
 
     file.print(*counter);
 
+    file.close();
+}
+
+uint8_t read_gdo_security_from_flash(const char* filename) {
+
+    File file = LittleFS.open(filename, "r");
+
+    if (!file) {
+        RINFO("%s doesn't exist. creating...", filename);
+
+        // DEFAULT TO THE NEWER +2.0 SECURITY
+        uint8_t secType = 2;
+        write_gdo_security_to_flash(filename, &secType);
+        return 1;
+    }
+
+    uint8_t secType = file.parseInt();
+
+    file.close();
+
+    return secType;
+}
+
+void write_gdo_security_to_flash(const char *filename, uint8_t* secType) {
```

Comment:
This function is identical to `write_file_to_flash`.  Can we use a common function?

Reply:
yup.... added default value to the read code